indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -221,6 +221,8 @@ def _send_emails(self, form):
             email = make_email(to_list=registration.email, cc_list=form.cc_addresses.data, bcc_list=bcc,
                                from_address=form.from_address.data, template=template, html=True,
                                attachments=attachments)
+            signals.event.before_registration_email_send.send('email-registrants', email=email,
```

Comment:
Alternatively, we could avoid adding a new signal at all and instead re-use `before_notification_send` here?

Like this:

```python
signals.core.before_notification_send.send('registration-custom-email', email=email,
                                           registration=registration, form=form)
```

(I'd prefer this over adding a new signal!)

Reply:
Hi Adrian,
I don't care about the type of signal, just need to capture "a signal" to change the email's contents, so YES, the solution you propose suits me as well.
If ok for you, we can go with that:
raise "before_notification_send" in RHRegistrationEmailRegistrants before send_mail.

Reply:
Let's go for that then. Please amend your commit and force-push the updated version to your branch.

While you are at it, we usually try to use [this style](https://chris.beams.io/posts/git-commit/) for commit messages (in particular points 1-6), so if you could adapt your commit message accordingly that'd be great (otherwise I can do it when merging)

Reply:
Done, tried to follow indication for commit message, hope that'll be fine.

Reply:
Yep, thanks!